main/lsp/lsp.h

```diff
@@ -171,7 +171,8 @@ class LSPLoop {
                          const std::vector<core::FileHash> &hashes) const;
     /** Apply conservative heuristics to see if we can run a fast path, if not, bail out and run slowPath */
     TypecheckRun tryFastPath(std::unique_ptr<core::GlobalState> gs,
-                             std::vector<std::shared_ptr<core::File>> &changedFiles, bool allFiles = false);
+                             const std::vector<std::shared_ptr<core::File>> &changedFiles,
```

Comment:
I made this parameter `const`, removed the now-unused `allFiles` parameter, and added an extra parameters for files that haven't changed but should be included on the fast path (which avoids needless copying and computation on unchanged files).